src/smc_sagews/smc_sagews/test_without_server/README.md

```diff
@@ -0,0 +1,57 @@
+# Unit test sagews without sage_server
+
+## Goals
+
+- test components of sage worksheets
+- avoid overhead and extraneous issues raised when testing through sage_server
+
+## How to Use These Tests
+
+### Running the Tests
+
+```
+cd cocalc/src/smc_sagews/smc_sagews/test_without_server
+
+# test jupyter_client launch of all 15 non-sage kernels
+python -m pytest
+
+# test selected kernels
+python -m pytest --kname=anaconda3,singular
+
+# look for failed tests
+grep fail ~/jclient-test-report.prom
+
+# output:
+# sagews_jclient_test{name="compute[apache_toree_scala]",outcome="failed"} 16.109400034 1496753157000
```

Comment:
just a silly detail, but here I don't know where this `compute[...]` is coming from.